### ISSUE TITLE
Easy installation YouTube guide removed

### ISSUE THREAD

**Issue by amitkeret** [aussieaddons/repo#91]
Self explanatory.
See at https://aussieaddons.com/installation/

**Comment by andybotting** [comment]
Yeah, YouTube recently removed it, citing that is was aiding piracy.

I'll have to find another way to host it.

**Comment by Corey-Evans** [comment]
Can you please just post the URL to add source in Kodi to save having to manually repo zip? Video probably isn’t really required.

**Comment by SamLeatherdale** [comment]
Yeah, struggling to find the repository URL. Can someone post it?

**Comment by gzuaps** [comment]
This should help
https://r.aussieaddons.com/

**Comment by SamLeatherdale** [comment]
Thanks!

I think it's also worth noting the install instructions:

- Add https://r.aussieaddons.com as a source in File Manager
- Addons > Install from Zip > Browse to your new Aussie Addons source and choose `repo-<version>.zip`
- Now the repo should be installed. Browse Addons again and choose the newly added Aussie Addons repo.
